cpp/src/parquet/properties.h

```diff
@@ -59,31 +59,32 @@ struct ParquetVersion {
 /// DataPageV2 at all.
 enum class ParquetDataPageVersion { V1, V2 };
 
-static int64_t DEFAULT_BUFFER_SIZE = 1024;
-static bool DEFAULT_USE_BUFFERED_STREAM = false;
+/// Align the default buffer size to the page size of most platforms.
+constexpr int64_t kDefaultBufferSize = 4096;
```

Comment:
You can bump this to 16384 for slightly better perf. 4096 is really too small nowadays.